chromium_src/sql/sql_name_variants.h

```diff
@@ -0,0 +1,21 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef SQL_SQL_NAME_VARIANTS_H_
+#define SQL_SQL_NAME_VARIANTS_H_
+
+#define IsValidDatabaseTag IsValidDatabaseTag_ChomiumImpl
+#include "../gen/sql/sql_name_variants.h"  // IWYU pragma: export
+#undef IsValidDatabaseTag
+
+namespace sql_metrics {
+
+constexpr bool IsValidDatabaseTag(std::string_view s) {
+  return true;
```

Comment:
nit: a comment would be nice here to mention that we don't care about `histograms.xml` sqlite tag sync as we don't have server side to match those.

Reply:
Added a comment in https://github.com/brave/brave-core/pull/26537/commits/4c851e683c8d5489c77720f66f8b67357907ab15